tests/cases/compiler/jsxEmptyExpressionNotCountedAsChild.tsx

```diff
@@ -0,0 +1,19 @@
+// @jsx: react,react-jsx,react-jsxdev

```

Comment:
are react/react-jsx/react-jsxdev meaningfully different for this change? Or does this just help coverage some other way? 

It was harder to read the baselines since they were in triplicate.

Reply:
Yep! The JS output is different for each! In `react` `createElement` is used, with arrays or singleton children. In `react-jsx`, the `jsx` and `jsxs` ctors are toggled between based on if the child is detected as singleton or not. In `react-jsxdev`, the `isStatic` parameter passed to the ctor is changed based on weather a singleton child is detected or not. So there's meaningful output to check for all 3!